src/Models/PlayerCardModel.php

```diff
@@ -53,22 +53,23 @@ public function getPlayerCardById(int $id): PlayerCard|false
         return $Player;
     }
 
-    public function getAllCards($DeleteStatus): array
+    public function getAllCards($DeleteStatus, $PositionId = ""): array
```

Comment:
Add type hints - I would actually suggest that you make deleteStatus a boolean instead of 0 or 1.

Reason being is that it's clearer to use, deleted = true means show me the deleted stuff etc

Reply:
You should also default position id to false